cyhy_report/customer/graphs.py

```diff
@@ -137,6 +137,7 @@ def plot(self, filename, size=1.0):
         majorLocator = MaxNLocator(nbins=5, integer=True)  # only mark integers
         ax.xaxis.set_major_locator(majorLocator)
 
+        ax.tick_params(axis="y", which="both", length=0)
```

Comment:
I think this could use a comment explaining why we're doing this.

Reply:
Good call - thanks for mentioning!  See https://github.com/cisagov/cyhy-reports/pull/54/commits/d637ed4590121cabb0c53a89236af930c481fe60.